feat(changelog): console-added-webauthn-based-1fa 2026-07-31 (#6749)
scaleway/docs-content: changelog/july2026/2026-07-31-console-added-webauthn-based-1fa.mdx

---
title: Webauthn based 1FA
status: added
date: 2026-07-31
category: console
product: console
---

> The Scaleway console now provides WebAuthn (Passkey) support for First-Factor Authentication (1FA) as complete single-step login that handles both factors at once.

Reviewer: ```suggestion
The Scaleway console now supports passkey (WebAuthn) authentication. If your passkey supports user verification, it acts as a passwordless login method, combining both authentication factors into a single gesture. Otherwise, it acts as a second factor alongside your password.

Passkeys are currently available for account Owners only, not IAM Members.
```

Reviewer: Je pense que la phrase n'est pas précise. Je propose donc cette modification.

> Secure your account via the [Account Security page](https://console.scaleway.com/settings/account/security) in your Settings space.

Reviewer: ```suggestion
Secure your account from the [Account Security](https://console.scaleway.com/settings/account/security) page in your **Settings** space.
```

> Read our [documentation on passkey authentication](https://www.scaleway.com/en/docs/account/how-to/use-2fa/#passkey-authentication) to learn more.

Reviewer: ```suggestion
See the [Passkey authentication](https://www.scaleway.com/en/docs/account/how-to/use-2fa/#passkey-authentication) documentation for more information.
```
